docs/src/tutorials/tutorial_6.md

```diff
@@ -0,0 +1,313 @@
+If I add something called `_anim` you can put it directly after `rotate_anim`. I'll otherwise only change the `actions` array.
+
+Our new animations:
+
+One translating from the origin to the right 
+
+```julia
+translate_anim = Animation(
+    [0, 1], # must go from 0 to 1
+    [O, Point(150, 0)],
+    [sineio()],
+)
+```
+
+and one translating back. One needs to make sure that it always starts at the origin (the current origin of the already translated view) to not introduce a break in the animation. More about that later.
+
+```julia
+translate_back_anim = Animation(
+    [0, 1], # must go from 0 to 1
+    [O, Point(-150, 0)],
+    [sineio()],
+)
+```
+
+and our `actions` (yes it's still a single [`Action`](@ref) but we add more soon)
+
+```julia
+actions = [
+    Action(
+        (args...) -> circle(O, 20, :fill);
+        subactions = [
+            SubAction(1:50, translate_anim, translate()),
+            SubAction(51:150, rotate_anim, rotate_around(Point(-150, 0))),
+            SubAction(151:200, translate_back_anim, translate()),
+        ],
+    )
+]
+```
+
+I changed the starting position of the circle to `O` and we now have three subactions.
+
+Let's walk through the steps Javis takes to create the animation:
+- For the first 50 frames of the action the circle is translated from `O` to `(150, 0)` so just to the right. 
+  - Remember the `O` is the same as `(0, 0)` and is at the center of the canvas at the beginning.
+  - Then the circle is at position `(150, 0)`. Remember that it is still drawn at the origin so actually we shifted our whole view to the right.
+- Therefore for the next 100 frames we need to specify that we actually want to rotate around the world origin which is at `Point(-150, 0)` from our current perspective. 
+- Afterwards we are now again ath `(150, 0)` but see it as our origin and therefore need to move our circle to the left to `Point(-150, 0)` which is the world origin. 
+
+Hope that makes sense! Let's see it in action:
+
+![The loading movement](assets/loading_movement.gif)
+
+## Adding more Blobs!
+
+The blob does start to feel lonely a bit so let's give him some friends. They should all do the same movement but start at different times.
+
+```julia
+actions = [
+    Action(
+        frame_start:frame_start+149,
+        (args...) -> circle(O, 20, :fill);
+        subactions = [
+            SubAction(1:30, translate_anim, translate()),
+            SubAction(31:120, rotate_anim, rotate_around(Point(-150, 0))),
+            SubAction(121:150, translate_back_anim, translate()),
+        ],
+    ) for frame_start in 1:10:50
+]
+```
+
+I've basically added four more blobs by using `for frame_start in 1:10:50` inside the array definition.
+Then I defined the start of the `Action` with: `frame_start:frame_start+149` such that every blob is there for 150 frames and reduced the number of frames in the subactions a bit to have 150 frames.
```

Comment:
At first, this syntax was not immediately apparent as a list comprehension. Really cool thing here! Maybe explicitly state this is a list comprehension being generated by the for loop?

Reply:
Added a small sentence! I use this quite often in my animations now as it makes everything quite simple